src/grafana_agent.py

```diff
@@ -402,7 +402,7 @@ def _loki_config(self) -> dict:
                 }
             )
 
-        if self.is_machine:
+        if self.is_machine():
```

Comment:
Minor stylistic preference (but also consistency with other similar methods in other o11y charms): I'd fix this by making `is_machine` a property.